doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -4008,8 +4106,21 @@ definition.
       rewrite /=.
       unlock lid.
 
-.. tacn:: unlock {? @occ_switch } @ident
-   :name: unlock
+.. tacn:: unlock {* {? %{ @clear_switch %} } @term } {? @ssrclauses }
+   :name: unlock (ssreflect)
+
+   .. insertprodn ssrclauses lcpattern
+
+   .. prodn::
+      ssrclauses ::= in @ssrclausehyps {? %|- } {? * }
```

Comment:
This non terminal should be suggestive about the fact that is describes where things are done. Does `clauses` work for that? IMO `ssrin` is better. BTW vanilla Coq has a `in ...` as well (in some tactics). Maybe we can tale inspiration there?

Reply:
I think I'd lean toward `ssrin` or `ssr_in`.  All nonterminals in Sphinx are global symbols, so there's a bit of juggling to prevent conflicts/confusiong between SSR, Ltac2 and vanilla nonterminals.

BTW, for readability, I'd suggest generally starting SSR nonterminals with `ssr_` (e.g. `ssr_XXX` rather than just `ssrXXX`).  I'd make that change in the mlg in a separate PR.

Reply:
Ok